src/Github/Domain/Value/CheckRun.php

```diff
@@ -92,6 +92,10 @@ public static function fromResponse(array $response): self
 
     public function isSuccessful(): bool
     {
+        if ('PHP 8.0 + highest + normal' === $this->name) {
```

Comment:
Take into account that when we start making 8.0 mandatory builds this won't be correct anymore

Reply:
Its easier to have this for now to get the dev-kit PRs merged.

I updated https://github.com/sonata-project/dev-kit/issues/1207 to remember to remove it!